File: src/main/kotlin/com/meow/meowchatting/user/repository/UserProfileRepository.java
package com.meow.meowchatting.user.repository;

import com.meow.meowchatting.user.command.domain.UserProfile;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface UserProfileRepository extends JpaRepository<UserProfile, Long> {
    Optional<UserProfile> findByUserId(Long userId);
}

File: src/main/kotlin/com/meow/meowchatting/service/user/repository/UserProfileRepository.kt
package com.meow.meowchatting.service.user.repository

import com.meow.meowchatting.service.user.domain.UserProfile
import org.springframework.data.jpa.repository.JpaRepository

interface UserProfileRepository: JpaRepository<UserProfile, Long> {
    fun findByUserId(userId: Long): UserProfile?
}
